examples/gltf/app.js

```diff
@@ -203,22 +215,32 @@ export class DemoApp {
   }
 
   onRender({gl, time, width, height, aspect}) {
-    gl.viewport(0, 0, width, height);
     clear(gl, {color: [0.2, 0.2, 0.2, 1.0], depth: true});
 
+    this.vrDisplay.getFrameData(this.frameData);
+
+    gl.viewport(0, 0, width * 0.5, height);
```

Comment:
Wonder if we should integrate this support into the `AnimationLoop` (so that render gets called twice per frame).

Reply:
I think this could be done pretty simply and generically, e.g. pass a `numPasses` argument (defaults to 1) to the `AnimationLoop` constructor which tells it how many times `onRender` should be called. Then the current pass index would be passed to `onRender` so it modify the draw commands. 

Could be generally useful for cases where you want to draw the whole scene more than once, e.g. shadow maps or HUD displays.